### PR TITLE
Add ES queries and ES index to all edges

### PR DESCRIPTION
**Pull request by berggren**
Fixes: #455 and #454

### CODE REVIEW

File: timesketch/lib/experimental/create_graph.cql [hunk 1]
```diff
@@ -32,6 +32,8 @@ WHERE line.src <> line.dst
   MERGE (src)-[r:ACCESS]->(dst)
```
Comment (franekp) [inline timesketch/lib/experimental/create_graph.cql]: What about adding es_index to nodes also?

File: timesketch/lib/experimental/win_services.py [hunk 2]
```diff
@@ -69,18 +69,22 @@ def win_services(sketch_id):
             sketch_id=sketch_id, query=u'event_identifier:7045'):
         data = event[u'_source'][u'xml_string']
         res = parse_xml(data)
+        res.extend((event.get(u'_index'), event.get(u'_id')))
```
Comment (franekp) [inline timesketch/lib/experimental/win_services.py]: This will create multiple edges for one service started multiple times, right?

Comment (berggren) [reply timesketch/lib/experimental/win_services.py]: It will in the output of this method, but will be merged when cypher adds them to the graph.

Comment (franekp) [reply timesketch/lib/experimental/win_services.py]: Sounds good.

File: timesketch/lib/experimental/create_graph.cql (hunk 1 above)
Comment (berggren) [reply timesketch/lib/experimental/create_graph.cql]: Done.

**Comment by berggren** [comment]
PTAL

**Pull request merged**